monitoring/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+export PORT=${PORT:-3001}
+export PORT2=${PORT2:-3002}
```

Comment:
It looks like we need to add support in docker-compose for at least 2 workers, so that user could:
1. Test and see how he can switch between workers in garafana
2. Lift up 2 workers with one docker-compose.
3. To have .env.example already pre-filled with this configuration
4. To have different volumes for both workers in docker-compose.yml